### PR TITLE
fix(crux-ui): add diff box-shadow and improve minor stylistic issues

### PR DESCRIPTION
**Pull request by polaroi8d**
The pull request contains the following minor changes:
- Add .vscode/*.log to gitignore file
- Remove unused label in the dyo-checkbox component
- Edit some text on the UI
- Improve box-shadow on models
- Remove unused IngressConfig from Agent
- Change magic hex colors to project defined ones

### CODE REVIEW

File: web/crux-ui/tailwind.config.js [hunk 1]
```diff
@@ -41,5 +44,5 @@ module.exports = {
     },
     plugins: [
         require('@tailwindcss/line-clamp'),
-    ],
+    ]
 }
```
Comment (polaroi8d) [inline web/crux-ui/tailwind.config.js:48]: NW at EOF I'll fix in the next commit.

File: web/crux-ui/src/elements/dyo-card.tsx [hunk 2]
```diff
@@ -4,8 +4,11 @@ import { RefAttributes } from 'react'
 export interface DyoCardProps extends RefAttributes<HTMLDivElement> {
   className?: string
   children: React.ReactNode
+  modal?: boolean
 }
 
 export const DyoCard = (props: DyoCardProps) => {
-  return <div className={clsx(props.className ?? 'p-8', 'card rounded-lg shadow-lg bg-medium')}>{props.children}</div>
+  const boxShadow = props.modal ? 'shadow-modal' : 'shadow-lg'
```
Comment (m8vago) [inline web/crux-ui/src/elements/dyo-card.tsx:11]: we don't need to lift this out from clsx

File: web/crux-ui/src/elements/dyo-modal.tsx [hunk 3]
```diff
@@ -34,7 +34,7 @@ const DyoModal = (props: DyoModalProps) => {
       >
         <Dialog.Overlay />
 
-        <DyoCard className={clsx(props.className, 'flex flex-col m-auto p-8')}>
+        <DyoCard className={clsx(props.className, 'flex flex-col m-auto p-8')} modal={true}>
```
Comment (m8vago) [inline web/crux-ui/src/elements/dyo-modal.tsx:37]: `modal` is enough

**Review by m8vago — CHANGES_REQUESTED** [review]

File: web/crux-ui/src/elements/dyo-card.tsx (hunk 2 above)
Comment (polaroi8d) [reply web/crux-ui/src/elements/dyo-card.tsx:11]: I thought it was more readable, you think I have to add back in the clsx func?

File: web/crux-ui/src/elements/dyo-modal.tsx (hunk 3 above)
Comment (polaroi8d) [reply web/crux-ui/src/elements/dyo-modal.tsx:37]: The reason why I am not an FE dev, thanks the point out!

**Review by robot9706 — APPROVED** [review]
😎 after m8vago's comments

**Review by m8vago — APPROVED** [review]
good job sir

**Pull request merged**